File: Count type of Characters/Count type of Characters.py
s="AZDVAIHsoasaiocha1231314)!@#!"
upper=0
lower=0
numbers=0
special=0
for i in s:
    if i.isupper():
        upper+=1
    elif i.islower():
        lower+=1
    elif i.isnumeric():
        numbers+=1
    else:
        special+=1
print(upper,lower,numbers,special)

File: Count type of Characters/Count type of Characters.cpp
#include <iostream>
#include <string>

using namespace std;

int main(){
int upper =0;
int lower = 0;
int numbers = 0;
int special = 0;


for (int i=0 ; i<s.length();i++){
    if (s[i] >= 'A' && s[i] <= 'Z'){
        upper++;
    }
    else if(s[i] >= 'a' && s[i] <= 'z'){
        lower++;
    }
    else if (s[i] >= '0' && s[i] <= '9'){
        numbers++;
    }
    else {
        special++;
    }
}
cout<<upper<<lower<<numbers<<special;


}